src/client/Microsoft.Identity.Client/AuthenticationResultMetadata.cs

```diff
@@ -77,6 +77,11 @@ public AuthenticationResultMetadata(TokenSource tokenSource)
         /// </summary>
         public RegionDetails RegionDetails { get; set; }
 
+        /// <summary>
+        /// Time, in milliseconds, spent in the token creation of the constrained delegation token.
+        /// </summary>
+        public long DurationInCdtInMs { get; set; }
```

Comment:
This is not a good name. MSAL should not know about "cdt". 

I think you want something like `DurationFormattingResult` or something like that

Reply:
Yes, we are going to discuss telemetry in our meeting tomorrow.